RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.cc

```diff
@@ -0,0 +1,115 @@
+#include "RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Utilities/interface/do_nothing_deleter.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include <iostream>
+#include <memory>
+#include <string>
+
+using namespace edm;
+
+OnlineBeamSpotESProducer::OnlineBeamSpotESProducer(const edm::ParameterSet& p) {
+  auto cc = setWhatProduced(this);
+
+  //transientBS_ = new BeamSpotObjects;
+  theHLTBS_ = new BeamSpotOnlineObjects;
+  theLegacyBS_ = new BeamSpotOnlineObjects;
+  fakeBS_ = new BeamSpotOnlineObjects;
+  fakeBS_->SetBeamWidthX(0.1);
+  fakeBS_->SetBeamWidthY(0.1);
+  fakeBS_->SetSigmaZ(15.);
+  fakeBS_->SetPosition(0., 0., 0.);
+  fakeBS_->SetType(-1);
+
+  bsHLTToken_ = cc.consumesFrom<BeamSpotOnlineObjects, BeamSpotOnlineHLTObjectsRcd>();
+  bsLegacyToken_ = cc.consumesFrom<BeamSpotOnlineObjects, BeamSpotOnlineLegacyObjectsRcd>();
+}
+
+void OnlineBeamSpotESProducer::fillDescription(edm::ConfigurationDescriptions& desc) {
```

Comment:
This should be
```suggestion
void OnlineBeamSpotESProducer::fillDescriptions(edm::ConfigurationDescriptions& desc) {
```
to have any effect.